doc/verification.rst

```diff
@@ -0,0 +1,106 @@
+Overview
+--------
+
+This is a SV/UVM testbench for the ibex core verification. This
+testbench loads the instruction binary generated from open source random
+instruction generator `riscv-dv`_, runs the RTL simulation, and compares
+the instruction trace against ISS simulation.
+
+Testbench component
+~~~~~~~~~~~~~~~~~~~
+
+-  Random instruction generator : `RISCV-DV`_
+-  `Memory interface agent for instruction fetch and load/store
+   operations`_
+-  `Interrupt interface agent`_
+-  `Memory model`_
+-  `Test and sequence library`_
+
+Testplan
+~~~~~~~~
+
+The goal of this bench is to fully verify the ibex core with 100%
+coverage. This includes all RV32IMC instructions testing, privileged
+spec compliance, exception and interrupt testing, debug mode operations
+etc. The complete test list can be found `here`_.
+
+Please note that this work is still working in progress.
+
+Getting Started
+---------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+-  VCS RTL simulator (need to support UVM 1.2)
+-  `Setup the RISC-V instruction generator and ISS sim environment`_
+
+End-to-end RTL/ISS co-simulation flow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. figure:: flow.png
```

Comment:
- You need to give the relative path to this image, that's images/flow.png. 
- Do you have a SVG version of this figure around? This will make editing easier in the future without having to redraw it.
- Can you rename the image to something like `dv-flow` to make it less generic?